src/pages/Movies.js

```diff
@@ -20,6 +20,7 @@ export default function Movies({
     </Wrapper>
   );
 }
+
 const Wrapper = styled.div`
   margin-top: 70px;
   margin-bottom: 68px;
```

Comment:
You could do that in one line here :)
```suggestion
  margin: 70px 0 68px;
```